src/main.jsx

```diff
@@ -1,18 +1,34 @@
-import * as React from 'react'
-import * as ReactDOM from 'react-dom/client'
+import React from 'react'
+import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
 import { createBrowserRouter, RouterProvider } from 'react-router-dom'
 import Home from './Components/Home.jsx'
-
 import './index.css'
 
+// const router = createBrowserRouter([
+//   {
+//     path: '/',
+//     element: <App>{ Home }</App> 
+//   },
+// ])
+
 const router = createBrowserRouter([
   {
     path: '/',
-    element: <App>{ Home }</App> 
+    element: <App />,
+    children: [
+      {
+        path: '/',
+        element: <Home />
+      }
+    ]
   },
```

Comment:
I initially had it like this but second guessed myself. This makes sense and won't have to become more complex as we won't have any other pages to route to.